app/views/inventories/show.html.erb

```diff
@@ -0,0 +1,21 @@
+<div class="inventory-show-page-main-container">
+  <div class="inventory-show-page-sub-container">
+		<h1><%= @inventory.name %></h1>
+		<%= link_to 'Add Food', new_inventory_inventory_food_path(@inventory), class: 'add-food-button' %>
+
+		<ul class="inventory-foods-container">
+			<li class="inventory-foods-header">
+        <span class="food-header"><strong>Food</strong></span>
+        <span class="quantity-header"><strong>Quantity</strong></span>
+        <span class="actions-header"><strong>Actions</strong></span>
+      </li>
+			<% @inventory_foods.includes(:food.name).each do |inventory_food| %>
+				<li class="inventory-foods-body">
+					<p class="quantity-body"><%= inventory_food.food.name %></p>
+					<p class="food-body"><%= inventory_food.quantity %></p>
+					<%= button_to "Delete", inventory_inventory_food_path(@inventory, inventory_food.id), method: :delete, class: "actions-body" %>
```

Comment:
- [ ] After logging out of the app and signing in with a different user, I am not able to see other users' inventory but only the one I create with the current user. Kindly make sure that a user is able to see all created inventories but can only remove the ones that he owns only.
- [ ] Also kindly make sure that the inventory details page matches the wireframe shown below. It is difficult to see whether your own matches it or not since the food is not populated here.


![Screenshot from 2023-08-20 01-13-27](https://github.com/sameedmohsin28/Recipe-App/assets/65233860/e18b482e-a3d0-4b8a-a3fd-939c55b162ce)

Reply:
![image](https://github.com/sameedmohsin28/Recipe-App/assets/122236545/3b020804-2c89-41e9-b85a-3a3fd2adf572)

- It says to display the list of inventories created by the logged in user in point 1